scripts/serve-laravel.sh

```diff
@@ -2,15 +2,17 @@
 
 mkdir /etc/nginx/ssl 2>/dev/null
 
+safe_1=${1//\ /_}
```

Comment:
IMHO the filename should be the primary server name. Especially not all of them since that would allow a user to list to many domains and cause the filename to be too long and fail.

Any other server names shouldn't be appended, they should be explicitly declared via config syntax (see previous comment).